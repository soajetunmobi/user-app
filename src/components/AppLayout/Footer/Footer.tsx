import React, { FunctionComponent } from 'react'

export const Footer: FunctionComponent = () => {
  const currentYear = new Date().getFullYear()

  return (
    <footer className='footer'>
      <span>User App</span>
      <br />
      {currentYear}
    </footer>
  )
}
