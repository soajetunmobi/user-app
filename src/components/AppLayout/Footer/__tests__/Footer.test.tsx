import React from 'react'
import renderer from 'react-test-renderer'
import { Footer } from '../Footer'

test('it renders correctly', () => {
  const root = renderer.create(<Footer />).toJSON()
  expect(root).toMatchSnapshot()
})
