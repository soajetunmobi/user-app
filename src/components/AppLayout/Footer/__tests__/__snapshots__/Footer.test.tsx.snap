// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<footer
  className="footer"
>
  <span>
    User App
  </span>
  <br />
  2020
</footer>
`;
